server/charm/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+name: hardware-api
+type: charm
+title: Hardware API Server
+summary: The Canonical Hardware API helps you determine if a given hardware platform is Ubuntu Certified.
+
+description: |
+  Canonical Hardware API Server is a RESTful API server that provides access to
+  hardware information for a given machine, in particular helping you determine if a given hardware platform is Ubuntu Certified,
+  matches partially to a Certified system, or is not Certified.
+
+  For more information, see https://github.com/canonical/hardware-api
+  For schema, see https://canonical.github.io/hardware-api
+
+containers:
+  hardware-api:
+    resource: hardware-api-image
+
```

Comment:
The build failure should be resolvable by removing the containers section from charmcraft.yaml